renderer/viewer/three/waypointSprite.ts

```
@@ -51,13 +53,21 @@ export function createWaypointSprite (options: {
   metadata?: any,
+  visualScale?: number,
 }): WaypointSprite {
   const color = options.color ?? 0xFF_00_00
   const depthTest = options.depthTest ?? false
   const labelYOffset = options.labelYOffset ?? 1.5
+  
+  // Get visual scale from options, metadata, server metadata, or default
+  // Priority: options.visualScale > metadata.visualScale > window.serverMetadata?.waypointVisualScale > DEFAULT
+  const visualScale = options.visualScale 
+    ?? options.metadata?.visualScale 
+    ?? (typeof window !== 'undefined' ? (window as any).serverMetadata?.waypointVisualScale : undefined)
```

ESLint

[error] Unexpected negated condition.

(unicorn/no-negated-condition)